noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -201,30 +201,9 @@ impl PrivateContext {
         self.nullifier_read_requests.push(request);
     }
 
-    pub fn request_app_nullifier_secret_key(&mut self, account: AztecAddress) -> Field {
+    pub fn request_nsk_app(&mut self, master_public_nullifying_key_hash: Field) -> Field {
```

Comment:
```suggestion
    pub fn request_nsk_app(&mut self, npk_m_hash: Field) -> Field {
```

Would keep the naming consistent here with notes ^ and also in other places. I have also introduced some naming incosistencies in other keys PR and I am fixing some of them in  my CompleteAddress PR. Would just follow the keys diagram as in noir there is no camel case issue.